### PR TITLE
Fix typos and misspellings found by CodeSpell 2.1.0.

### PR DESCRIPTION
**Pull request by warmsocks**
Also fix a few Python whitespace errors found by Syntastic.

### CODE REVIEW

File: src/python/Collection.py [hunk 1]
```diff
@@ -20,11 +20,12 @@
 
 from types import *
 
+
```
Comment (krop) [inline src/python/Collection.py:23]: Was the empty new line added by codespell? (here and on line 84)

**Comment by winterz** [comment]
This is great.  Thanks for letting me know about codespell.
Other than the 1 extra blank-line in src/python/Collection.py this patch is good to go.

**Comment by winterz** [comment]
oh, can you also add a .codespellrc (especially if you have 'skip' or 'ignore-words-list settings')

**Comment by winterz** [comment]
I'll merge this, then adjust the extra line and add a .codespellrc

**Pull request merged**